modules/javafx.graphics/src/main/java/com/sun/prism/es2/IOSGLDrawable.java

```diff
@@ -29,10 +29,10 @@
 class IOSGLDrawable extends GLDrawable {
 
     private static native long nCreateDrawable(long nativeWindow, long nativeCtxInfo);
+    private static native void nDestroyDrawable(long nativeCtxInfo);
```

Comment:
I would recommend to rename these method as `nReleaseDrawable`.
I see we use names as  ReleaseXxxx and DeleteXxxx. The DestroyXxxx name would be a new addition, I think we could avoid the variation.

Reply:
Renamed as suggested.